_posts/en/newsletters/2020-02-05-newsletter.md

```diff
@@ -0,0 +1,272 @@
+---
+title: 'Bitcoin Optech Newsletter #83'
+permalink: /en/newsletters/2020/02/05/
+name: 2020-02-05-newsletter
+slug: 2020-02-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the release of Eclair 0.3.3, requests
+help testing a Bitcoin Core maintenance release, links to a new tool for
+experimenting with taproot and tapscript, summarizes discussion about
+safely generating schnorr signatures with precomputed public keys, and
+describes a proposal for interactive construction of LN funding
+transactions.  Also included is our regular section about notable
+changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Upgrade to Eclair 0.3.3:** this new release includes support for
+  [multipath payments][topic multipath payments], deterministic builds
+  for eclair-core (see the [PR description][eclair1295] below), experimental support
+  for [trampoline payments][topic trampoline payments], and various minor
+  improvements and bug fixes.
+
+- **Help test Bitcoin Core 0.19.1rc1:** this upcoming maintenance
+  [release][bitcoin core 0.19.1] includes several bug fixes.
+  Experienced users are encouraged to help test for any regressions or
+  other unexpected behavior.
+
+## News
+
+- **Taproot and tapscript experimentation tool:** Karl-Johan Alm
+  [posted][alm btcdeb] to the Bitcoin-Dev list about an experimental branch of his
+  [btcdeb][] tool that supports creating and executing [taproot][topic
+  taproot] and [tapscript][topic tapscript] outputs using a command line
+  tool named `tap`.  For more information, see his detailed
+  [tutorial][tap tutorial].
+
+{% comment %}<!-- Timeline:
+1/26 15:29 gmaxwell
+https://github.com/bitcoin-core/secp258k1/pull/558#discussion_r371027220
+"shouldn't this present an interface that takes the pubkey as an
+argument,"
+
+1/26 16:05 jonasnick
+"Note that in that case we'll need to add the pubkey as extradata to the
+deterministic nonce function, otherwise calling sign with the wrong
+public key may leak the secret key through an (invalid) signature."
+
+1/27 18:36 #secp256k1
+[6:36:02 pm] <gmaxwell> but it's not really a problem in ed25519 because the only thing it does with the public key is stuffs it into the e hash, and presumably it stuffs the same data into the nonce hash?  (if it doesn't they're pants on head stupid and should be shot)
+[...]
+[6:37:49 pm] <gmaxwell> okay the ed25519 people should be shot, it's an input to e and not an input to their nonce function
+[6:38:02 pm] <gmaxwell> (I just checked)
+
+1/28 01:49 https://moderncrypto.org/mail-archive/curves/2020/001012.html
+-->{% endcomment %}
+
+- **Proposed amendments to BIP340 schnorr signatures:** [libsecp256k1 PR
```

Comment:
This write-up does a very good job at narrating the events in order (and attributing the authors). I wonder though if it could be made more concise by simply documenting the conclusions rather than replaying the messages in order (or perhaps leading with the conclusions, then providing more details for context). As it stands, it's a bit difficult to pick out the two conclusions:

- use evenness as the tiebreaker for x-only pubkeys
- don't use untrusted passed-in pubkey when signing

Reply:
I pushed an attempt to resolve this concern by splitting the two parts into separate bullets and giving each a concluding paragraph.

Reply:
I think this is much clearer as two bullets. Thanks!